core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -8772,6 +8772,64 @@ private void checkLiteral2(String expression, String expected) {
         .withOracle(11).ok(expectedVersionLow1);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-5583">[CALCITE-5583]
+   * Rel2Sql will get an error when select * and join is present</a>. */
+  @Test void testSelectStarWithJoinOn() {
+    final String sql = "SELECT * FROM \"DEPT\" t1 "
```

Comment:
Can this SQL be executed directly in PG?

Reply:
I verified that a similar example can run on a pgSQL web page. [test for pgsql](https://onecompiler.com/postgresql/43n9x23hm)

Reply:
Get it. Thanks.